### ISSUE TITLE
main.cfg resets due to unknown error in "toolleveling" section

### ISSUE THREAD

**Issue by Gunner76th** [SlimeKnights/IguanaTweaksTConstruct#133]
So I went through and compared both the errored file as well as the one that loaded fine, and aside from the number of levels I told it to allow, I cannot find a reason that the mod decided to replace the config file. What is worse is, it errored the config file, even though it was working just fine on our previous map.

Old Config file #1: http://pastebin.com/jQAEDttr << from previous map

Old Config file #2: http://pastebin.com/7ejmuvaP << Manually fixed and tried to use with new map

New Config file #1: http://pastebin.com/qmsSRxWp << All sections copied from Old Config #2 except for Toolleveling section. Server did not replace this file version.

New Config File #2: http://pastebin.com/39GQCUH7 << Updated Toolleveling section to set tool level max to 15.

Both New #1 and New #2 worked, which is odd seeing as from what I can tell New #2 is the same as Old #2.

**Comment by bonii-xx** [comment]
Was there an error in the log regarding this? Besides that it sounds like Forge itself couldn't read the file, making me suspect your file permissions might've been set incorrectly (no read/write access on the file?).

**Comment by Gunner76th** [comment]
nope file permissions were fine. I know this because I did not change or touch them on the one it generated the first time, and it still marked it as errored (Old #2 was one the mod generated, I edited, and then it said was bad).

As for the log, there is no crash report, and if you are referring to the forge server log, no I have not looked through the 50K lines to see if there is any mention from iguana tweaks about it, because I would have no idea what I would be looking for.
